portfolio/src/main/webapp/index.html

```diff
@@ -60,54 +60,136 @@ <h2>My Portfolio</h2>
-                        </div>
-                        <div class="button-container">
-                            <button>Back</button>
+                            <div class="button-container">
+                                <button onclick="flipCard(this)">Back</button>
+                            </div>
                         </div>
                     </div>
                     <div class="card">
-                        <div class="card-content">
-                            <img src="/images/cat2.jpg" />
+                        <div class="card-front">
+                            <div class="card-content">
+                                <img src="/images/cat1.jpg" />
+                            </div>
+                            <div class="button-container">
+                                <button onclick="flipCard(this)">More Details</button>
+                            </div>
                         </div>
-                        <div class="button-container">
-                            <button>More Details</button>
+                        <div class="card-back">
+                            <div class="card-content">
+                                <h3 class="card-title">Calculator</h3>
+                                <p class="card-details">Lorem Ipsum</p>
+                                <div class="card-labels-section">
+                                    <p>Tags:</p>
+                                    <div class="labels-row">
+                                        <div class="label">
+                                            <div class="label-semi-circle-part"></div>
+                                            <div class="label-main-part" style="background-color: green"></div>
+                                        </div>
+                                        <div class="label">
+                                            <div class="label-semi-circle-part"></div>
+                                            <div class="label-main-part" style="background-color: red"></div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="button-container">
+                                <button onclick="flipCard(this)">Back</button>
+                            </div>
                         </div>
                     </div>
                     <div class="card">
-                        <div class="card-content">
-                            <img src="/images/cat1.jpg" />
+                        <div class="card-front">
+                            <div class="card-content">
+                                <img src="/images/cat2.jpg" />
+                            </div>
+                            <div class="button-container">
+                                <button onclick="flipCard(this)">More Details</button>
```

Comment:
It still appears on a flipped card.

Reply:
I did a trick with angles and rotationY on CSS to handle the flipping. Actually, the "card-back" section is rotated 180deg and so it is facing the other side of the card.

Reply:
Yes, but the "MORE DETAILS" button is still visible and rotated 180 degrees